### ISSUE TITLE
Add HTTPS certificate

### ISSUE THREAD

**Issue by naltun** [rubyapi/rubyapi#144]
I recommend using [Let's Encrypt](https://letsencrypt.org/).

Let's consider:
![image](https://user-images.githubusercontent.com/7507990/62542516-e8c67180-b829-11e9-96e0-468ece780eb2.png)

![image](https://user-images.githubusercontent.com/7507990/62542528-ef54e900-b829-11e9-8994-af2d31c21889.png)

This is important so that users can access the site through a certified secure connection.

Let's Encrypt is perfect for this (they're a great NGO) and it is easy to set up.

BTW: great project!

**Comment by colby-swandale** [comment]
👋 All traffic to Ruby API is being served through Cloudflare with HTTPS set to required, so I'm not sure what's going on here.

Can you provide some information/debug about the certificate that's being given to your browser?

**Comment by naltun** [comment]
Thanks for getting back swiftly 🙏

Is there even a certificate then? Is it provided through Cloudflare, or is it using a self-signed cert? Or, in this case, would Cloudflare being the certificate holder?

I am behind a firewall, but the rules allow for any secure site to serve resources. I am not sure what's going on, also.

**Comment by colby-swandale** [comment]
There is defiantly a signed and valid certificate that Cloudflare is serving to visitors. Are you able to try on a mobile device or another browser to check if this problem is the same?
 
<img width="1552" alt="Screen Shot 2019-08-08 at 12 02 48 am" src="https://user-images.githubusercontent.com/996377/62629162-00a80d80-b970-11e9-86b0-a761353ffef7.png">

**Comment by naltun** [comment]
Working on mobile.

In fact, I just revisited the site on my laptop and I am now getting a 'blocked access - Unrated' redirect landing page.

I suppose I triggered the VPN to register the site, and the rating is pending...

The VPN is FortiGuard, btw. It's used in corporate environments. I don't think there's anything to worry about, and the filter will probably go away once it's rated. rubyapi is using a valid cert, correct? And it's behind Cloudflare. I think the restriction will be lifted for me soon enough.

This issue can be considered resolved. Thanks for being patient!.
